deployer/aws-lambda/content-origin-request/index.js

```diff
@@ -6,7 +6,16 @@ const {
   encodePath,
   slugToFolder,
 } = require("@yari-internal/slug-utils");
+const { VALID_LOCALES } = require("@yari-internal/constants");
 
+const ONE_MONTH = 3600 * 24 * 30;
+// Note that the keys of "VALID_LOCALES" are lowercase locales.
+const LOCALE_URI_WITHOUT_TRAILING_SLASH = new Set(
+  [...VALID_LOCALES.keys()].map((locale) => `/${locale}`)
+);
+const SEARCH_URI_WITH_TRAILING_SLASH = new Set(
+  [...VALID_LOCALES.keys()].map((locale) => `/${locale}/search/`)
+);
```

Comment:
Soon we're also going to have 
- `/en-US/signin`
- `/en-US/signup`
- `/en-US/settings`
- `/en-US/next-cool-thing`

So basically the **only** time we want to force in a trailing slash is when the URL is *just* a locale code.